### DISCUSSION TITLE
Why do I get so many errors?

### DISCUSSION THREAD

**Discussion by Naadiyaar** [mfussenegger/nvim-jdtls#682] [Q&A]
Here I've downloaded JakartaEE-mail source code to study it locally, but nvim-jdtls is showing tons of different errors and warning. However the project compiles with no issue. Any idea what's the problem?
screenshot from `mail-api/api/src/main/java/jakarta/mail/Transport.java` :
![Screenshot from 2024-08-14 21-33-16](https://github.com/user-attachments/assets/cfa9677e-15c1-42c2-bdce-dfbe28cbdcc1)

Link to project:
https://github.com/jakartaee/mail-api/tree/master

Edit: it worth mentioning that the plugin is working fine in general and I'm facing this issue only with this project

**Comment by rogeriofrsouza** [comment]
I've seem these errors as well.
I don't know much about it, seems like multi-modules projects requires some LSP config.
Btw, the new [Eclipse Angus Mail project ](https://github.com/eclipse-ee4j/angus-mail) doesn't have these same errors.
